protobufs/livekit_sip.proto

```diff
@@ -47,8 +47,13 @@ option ruby_package = "LiveKit::Proto";
 
 
 service SIP {
-  rpc CreateSIPTrunk(CreateSIPTrunkRequest) returns (SIPTrunkInfo);
-  rpc ListSIPTrunk(ListSIPTrunkRequest) returns (ListSIPTrunkResponse);
+  rpc CreateSIPTrunk(CreateSIPTrunkRequest) returns (SIPTrunkInfo) { option deprecated = true; };
+  rpc ListSIPTrunk(ListSIPTrunkRequest) returns (ListSIPTrunkResponse) { option deprecated = true; };
```

Comment:
what should the behavior be for a deprecated trunk? should it continue to return all trunks, and indicating the type? or should it only list trunks are created with the old API?

Reply:
My plan was that the list will still return old trunks + all new ones (converted with the helpers). So the user will see correct metadata, but the config for them may be incomplete (e.g. only the first `number` visible).